src/app/common/schemas.py

```diff
@@ -59,19 +86,52 @@ class FormatTypes(str, Enum):
 
 
 class BoundingBox(BaseModel):
-    """Bounding box schema."""
+    """Bounding box schema for defining a rectangular area within an image.
 
-    x0: float = Field(..., example=0.0)
-    y0: float = Field(..., example=0.0)
-    x1: float = Field(..., example=100.0)
-    y1: float = Field(..., example=100.0)
+    This schema represents the coordinates of the box’s corners, which can be used
+    to specify an area of interest in image processing tasks. Coordinates are
+    defined with the origin at the top-left of the image.
```

Comment:
maybe add that the coordinates are defined in pixels?

Reply:
I like that idea.